tests/model_formsets/test_uuid.py

```diff
@@ -91,3 +91,27 @@ def test_inlineformset_factory_nulls_default_pks_alternate_key_relation(self):
         )
         formset = FormSet()
         self.assertIsNone(formset.forms[0].fields["parent"].initial)
+
+    def test_inlineformset_factory_nulls_default_pks_alternate_key_with_child(self):
+        """
+        After validate formset, Parent object with a UUID alternate key
+        should not None.
+        """
+        FormSet = inlineformset_factory(
+            ParentWithUUIDAlternateKey, ChildRelatedViaAK, fields="__all__"
+        )
+        formset = FormSet(
+            {
+                "childrelatedviaak_set-TOTAL_FORMS": 3,
+                "childrelatedviaak_set-INITIAL_FORMS": 0,
+                "childrelatedviaak_set-MAX_NUM_FORMS": "",
+                "childrelatedviaak_set-0-name": "Foo",
+                "childrelatedviaak_set-1-name": "",
+                "childrelatedviaak_set-2-name": "",
+            }
+        )
+        self.assertIs(formset.is_valid(), True)
+        # Parent object key in not None after validate.
+        self.assertIsNotNone(formset.instance.uuid)
+        # Parent object key and child related key have same value
```

Comment:
This comment is also unnecessary, it describes only an assertion without more explanation.
```suggestion
```